lapack.go

```diff
@@ -136,3 +136,23 @@ const (
 	InitZ   Comp = 'I'
 	UpdateZ Comp = 'V'
 )
+
+// EigVecSide specifies what eigenvectors will be computed.
+type EigVecSide byte
```

Comment:
Already settling on the names of this type, the one below, and of the constant values was a struggle. `blas.Side` cannot be used because it doesn't cover the `'B'` value and also the semantics is different. I'm very open to other suggestions.